server/src/main/java/io/crate/execution/engine/aggregation/impl/TopKAggregation.java

```diff
@@ -436,13 +470,41 @@ int id() {
         }
     }
 
+    private static boolean supportedByLongSketch(DataType<?> type) {
```

Comment:
If my comment of "all numeric types can be supported" is valid we can replace `supportedByLongSketch` with 
`NUMERIC_PRIMITIVE_TYPES.contains(type)`